tests/unit/components/Collections/ListItems/PlioListItem.spec.js

```diff
@@ -0,0 +1,31 @@
+import { mount } from "@vue/test-utils";
+import PlioListItem from "@/components/Collections/ListItems/PlioListItem";
+import Tooltip from "primevue/tooltip";
+
+// as window.matchMedia is not defined in the DOM
+Object.defineProperty(window, "matchMedia", {
```

Comment:
should this also be in `jest.init.js` like the other window property you've defined there?